backend/Dockerfile

```diff
@@ -86,12 +92,12 @@ FROM base as production
 # Copy "binary"-files from build image
```

Comment:
`test` stage is not necessary :point_up:

Reply:
test stage is used in https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/.github/workflows/test.yml#L72